src/marketing/email-template-list.md

```diff
@@ -17,31 +17,31 @@ The list of email templates is organized alphabetically by Magento module.
 {: .template-table }
 |Template|Configuration path|
 |--- |--- |
-| Assign Company Admin | **Page:** <br/>**Section:** <br/>**Field:** |
-| Assign Company to Customer | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Admin Changed to Member | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Admin Set Inactive | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Registration Request | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Status Active1 | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Status Active2 | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Status Blocked | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Status Pending Approval | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Status Rejected | **Page:** <br/>**Section:** <br/>**Field:** |
-| Customer Status Active | **Page:** <br/>**Section:** <br/>**Field:** |
-| Customer Status Inactive | **Page:** <br/>**Section:** <br/>**Field:** |
-| Sales Representative Assigned to Company | **Page:** <br/>**Section:** <br/>**Field:** |
+| Assign Company Admin | **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:**Customer-Related Emails<br/>**Field:** Default 'Assign Company Admin' Email|
+| Assign Company to Customer | **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:**Customer-Related Emails <br/>**Field:** Default 'Assign Company to Customer' Email|
+| Company Admin Changed to Member | **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:**Customer-Related Emails<br/>**Field:** Default 'Company Admin Changed To Member' Email|
+| Company Admin Set Inactive | **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:**Customer-Related Emails<br/>**Field:** Default 'Customer Status Inactive' Email|
+| Company Registration Request | **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:** Email Options - Company Registration<br/>**Field:** Default Company Registration Email|
+| Company Status Active1 | **Page:** **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:**Company Status Change<br/>**Field:** Default 'Company Status Change To Active 1" Emai|
```

Comment:
There are 5 double "Page:"s here.

Reply:
Thank you @dobooth,
I have fixed this!

Could you please approve?